db/migrate/20200318135632_rename_report_templates.rb

```diff
@@ -0,0 +1,30 @@
+class RenameReportTemplates < ActiveRecord::Migration[5.2]
+  TEMPLATE_NAMES = {
+    "Ansible Inventory" => "Ansible - Ansible Inventory",
+    "Applicable errata" => "Host - Applicable Errata",
+    "Applied Errata" => "Host - Applied Errata",
+    "Entitlements" => "Subscription - Entitlement Report",
+    "Host statuses" => "Host - Statuses",
+    "Registered hosts" => "Host - Registered Content Hosts",
+    "Registered users" => "User - Registered Users",
+    "Subscriptions" => "Subscription - General Report",
+  }
+
+  class FakeReportTemplate < ApplicationRecord
+    self.table_name = 'templates'
+  end
```

Comment:
Maybe this would be good idea, not to accidently override other template?
```suggestion
    default_scope where(:type => 'ReportTemplate')
  end
```

Reply:
I don't think so, this is all about avoiding all orchestration, validations, callbacks etc.

Reply:
But what if we delete other type of template???!
There are no callbacks and even on the real template class are just validations that are OK to run!
Apart of that, `default_scope` won't run any validations, it just makes sure we handle only templates we want to.

Reply:
not sure if you suggest to use real class however we should avoid that. The famous issue of model version does not match the schema version... e.g. in future we add a new column to SQL and will add a validation it must be present, for new installation the migration would start failing since the column does not exist yet however the validation would be already used

however the default scope should be fine, we limit the scope of templates we touch, which is imho good idea

Reply:
yeah, I was not really clear:
IMHO for template it wouldn't even hurt to use real class, if such unlikely situation happens, we know right away, because we test our migrations. I'm ok and in favor of using Fake class, but it need to be used with care as well in this case what sounds like fake model of `ReportTemplate` is actualy fake model of `Template`.
That said, we definitely need the `default_scope` IMO, if we would insist on not having one, I'd insist on renaming the model to `FakeTemplate` to at least be clear on the fact.

Reply:
yeah, I agree with using `default_scope` but kind of insist on using FakeModel in migration, it's very painful to fix such issues and they can be hard to detect on time, especially if we cherry-pick things to stable branches

Reply:
oh, right :) well, I agree on using `FakeModel` for sure :+1: